README.md

```diff
@@ -8,9 +8,14 @@ Derelict|Prisoner
 -|-
 ![](media/carousel.png) | ![](media/prisoner.png)
 
+# Build guide
+* Install the "renderdoc" library
```

Comment:
vkblam does not actually need the renderdoc library to be compiled as it is purely used for my development environment to debug stuff on the GPU. Currently only debug-builds on non-windows platforms technically require it to keep it out of the way of any binary releases as this is the primary development-environment.

Reply:
Okay I removed this from the README.md, should be ready for merge now then